base/replutil.jl

```diff
@@ -444,6 +456,10 @@ function show_method_candidates(io::IO, ex::MethodError, kwargs::Vector=Any[])
             buf = IOBuffer()
             tv = Any[]
             sig0 = method.sig
+            # @Hardcoded: don't display the sysimg.jl method
```

Comment:
Again, this it not the place to discuss this. Please open a separate issue if you have a proposal about this.